site/en/guide/tpu.ipynb

```diff
@@ -96,8 +93,9 @@
         "id": "yDWaRxSpwBN1"
       },
       "source": [
-        "## TPU Initialization\n",
-        "TPUs are usually on Cloud TPU workers which are different from the local process running the user python program. Thus some initialization work needs to be done to connect to the remote cluster and initialize TPUs. Note that the `tpu` argument to `TPUClusterResolver` is a special address just for Colab. In the case that you are running on Google Compute Engine (GCE), you should instead pass in the name of your CloudTPU."
+        "## TPU initialization\n",
+        "\n",
+        "TPUs are typically Cloud TPU workers, which are different from the local process running the user's Python program. Thus, you need to do some initialization work to connect to the remote cluster and initialize the TPUs. Note that the `tpu` argument to `TPUClusterResolver` is a special address just for Colab. If you are running your code on Google Compute Engine (GCE), you should instead pass in the name of your Cloud TPU."
```

Comment:
while you're here, can you add the full symbol name for `TPUClusterResolver` so we get the auto-link behavior?

Reply:
Done 👍